android-project/CMake/android_defs.cmake

```diff
@@ -15,3 +17,6 @@ set(TTF_FONT_DIR \"\")
 file(
   COPY "${DevilutionX_SOURCE_DIR}/Packaging/resources/CharisSILB.ttf"
   DESTINATION "${DevilutionX_SOURCE_DIR}/android-project/app/src/main/assets")
+
+file(GLOB VirtualGamepadArt "${DevilutionX_SOURCE_DIR}/Packaging/resources/ui_art/*")
+file(COPY ${VirtualGamepadArt} DESTINATION "${DevilutionX_SOURCE_DIR}/android-project/app/src/main/assets/ui_art")

```

Comment:
This feels like it soft of kind of depends on https://github.com/diasurgical/devilutionX/issues/2488

Reply:
Yeah, and the code to load these assets is actually using `SFileOpenFile()`. Having a build-time option to include these assets in the MPQ would be nice.